scripts/pugme.js

```diff
@@ -0,0 +1,41 @@
+// Description:
+//   Pugme is the most important thing in life
+//
+// Dependencies:
+//   None
+//
+// Configuration:
+//   None
+//
+// Commands:
+//   randbot pug me - Receive a pug
+//   randbot pug bomb N - get N pugs (max 10)
+
+module.exports = function (robot) {
+  robot.respond(/pug me/i, msg => msg.http('http://pugme.herokuapp.com/random').get()((err, res, body) => msg.send(JSON.parse(body).pug)));
+  robot.respond(/pug bomb( (\d+))?/i, (msg) => {
+    let count;
+    if (parseInt(msg.match[2]) > 10) {
+      count = 10;
+    }
+    else {
+      count = msg.match[2] || 5;
+    }
+    return msg.http(`http://pugme.herokuapp.com/bomb?count=${count}`).get()((err, res, body) => {
+      let i,
+        len,
+        pug,
+        ref,
+        results;
+      ref = JSON.parse(body).pugs;
+      results = [];
+      for (i = 0, len = ref.length; i < len; i++) {
+        pug = ref[i];
+        results.push(msg.send(pug));
+      }
+      return results;
+    });
+  });
+  return robot.respond(/how many pugs are there/i, msg =>
```

Comment:
Do we need this command? I don't think `robot.respond` returns anything worthwhile and we don't need to keep this command around. I don't know if I care how many pugs are there on somed Tumblr page haha.